holidayextras/node-database
# TECH-1122 adding a port option to dabase connection configuration

#### What does this PR do?

Adds the option to state which port you wish to connect on. This does not mean all projects that currently use this module will need to be updated to specify a port as leaving it blank should result in the use of the default port. 
#### What unit or integration tests does this PR have?

None.
#### What selenium tests does this PR have?

None.
#### How should a developer review this?

Look at the teeny-tiny change and check it seems reasonable. 
#### How should this be manually tested?

Go into the repo
`node test` to run the test, you should see a result.
If you insert `port: 3306,` or `port: undefined`, around line 6 of the test and run it you should see the same result.
#### Any background context you want to provide?

A project I'm working on needs to be able to specify the port it will connect to a database on, nuff said. 
#### What are the relevant tickets?

TECH-1122
#### Screenshots (if appropriate)
#### What gif best describes how you feel about this work?

![avenger dogs](http://www.reactiongifs.com/wp-content/uploads/2013/03/puppy-avengers.gif)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [x] I've checked there is appropriate unit, selenium and regression test coverage.
- [x] I've tested this cross-browser/device for visual changes.
- [x] I've manually checked this work against any dependent systems.
- [x] I've checked that there are no negative speed or performance impacts from my work.
- [x] I've added appropriate tracking.
#### IP or Developer Review:
#### Software Engineer or Developer review:
#### Software Engineer or project guru review:
#### Software Tester review
- [ ] I’ve run all the selenium tests locally and they pass.
- [ ] I agree with the test coverage.
- [ ] I've checked for Ruby changes and understand how to deploy if present.
